erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py

```diff
@@ -75,15 +81,6 @@ def add_suffix_if_duplicate(account_name, account_number, accounts):
 
 	return account_name, account_name_in_db
 
-def identify_is_group(child):
-	if child.get("is_group"):
-		is_group = child.get("is_group")
-	elif len(set(child.keys()) - set(["account_type", "root_type", "is_group", "tax_rate", "account_number"])):
-		is_group = 1
```

Comment:
@nextchamp-saqib, these are the lines I was talking about that misbehave in setting the `is_group` flag. With the new optional currency column addition, this may or may not match, and it will erratically set the group flag.